Makefile

```diff
@@ -38,13 +38,14 @@ download-portmap:
 
 # Default to build the Linux binary
 build-linux:
-	GOOS=linux CGO_ENABLED=0 go build -o aws-k8s-agent -ldflags "$(LDFLAGS)"
-	GOOS=linux CGO_ENABLED=0 go build -o aws-cni -ldflags "$(LDFLAGS)" ./plugins/routed-eni/
+	GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0 go build -o aws-k8s-agent -ldflags "$(LDFLAGS)"
+	GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0 go build -o aws-cni -ldflags "$(LDFLAGS)" ./plugins/routed-eni/
 
 docker-build:
-	docker run -v $(shell pwd):/usr/src/app/src/github.com/aws/amazon-vpc-cni-k8s \
+	docker run -v $(shell pwd):/usr/src/app/src/github.com/aws/amazon-vpc-cni-k8s:z \
```

Comment:
Why do we need the `:z` suffix for shared bind mounts? Is it just for building on SELinux?

Reply:
Exactly, it is just for building on SELinux. I put it as my machine is enabled SELinux (and sorry I forgot to remove it when I sent this PR). The option was removed now.

Reply:
Seems like it doesn't have any negative effects on non SE systems, so maybe we could have it around. I had no idea it was needed on SELinux, thanks for letting me know 🙃